datacats/environment.py

```diff
@@ -218,10 +221,27 @@ def load(cls, environment_name=None, data_only=False):
             always_prod = cp.getboolean('datacats', 'always_prod')
         except NoOptionError:
             always_prod = False
+
+        # if remote_server's custom ssh connection
+        # address is defined,
+        # we overwrite the default datacats.com one
         try:
-            deploy_target = cp.get('deploy', 'target', None)
-        except NoSectionError:
-            deploy_target = None
+            deploy_target = cp.get(
+                'deploy', 'remote_server_user', None) \
+                + "@" + cp.get('deploy', 'remote_server', None)
+        except:
+            deploy_target = DEFAULT_REMOTE_SERVER_TARGET
+
+        # if remote_server's ssh public key is given,
+        # we overwrite the default datacats.com one
+        try:
+            known_hosts_path = datadir + '/known_hosts'
```

Comment:
I've been sticking these sorts of temporary files in the "run" directory, although that's not perfect. Maybe we should be using proper temp files